Cpp/odin-views/app.facebook.cpp

```diff
@@ -105,27 +106,43 @@ namespace {
                 /// their Facebook a/c to their pre-existing identity.
                 identity_id = req.headers()["__user"].value();
                 app_user_id = req.headers()["__app_user"].value();
+                if (user_detail.has_key("email")) {
+                    auto const email_owner_id =
```

Comment:
I think this can be more specific, calling it `email_owner_id` may introduce confusion